src/editor/panels/TopPanel.js

```diff
@@ -968,7 +968,7 @@ class TopPanel {
        </div> <!-- a_panel -->
        <div id="path_node_panel">
          <div class="tool_sep"></div>
-         <se-button id="tool_node_link" title="${i18next.t('tools.node_link')}" src="./images/tool_node_link.svg" pressed>
+        <se-button id="tool_node_link" title="${i18next.t('tools.node_link')}" src="./images/tool_node_link.svg" pressed={${true}}>
```

Comment:
pressed is equivalent to pressed={$(true)} ??

Reply:
```
<div id="tool_node_link" class="tool_button push_button_pressed" title="Link Control Points"></div>
```

Initially `push_button_pressed` class added in  `tool_node_link` button, that why pressed set true.

icon also changed.